config/routes.rb

```diff
@@ -5,7 +5,9 @@
   end
   resource :checkout, only: [:create, :update]
 
-  get 'checkout', to: "checkouts#edit"
+  resources :orders, only: [:create]
+  get 'checkout/confirmation', to: "checkouts#confirmation"
+  get 'checkout', to: "checkouts#show"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.